pkgs/by-name/ll/lldap-cli/package.nix

```diff
@@ -0,0 +1,56 @@
+{ lib
+, stdenvNoCC
+, fetchFromGitHub
+
+, curl
+, jq
+, makeWrapper
+, lldap
+, util-linux
+}:
+
+stdenvNoCC.mkDerivation (finalAttrs: rec {
```

Comment:
Why is there a `rec` alongside `finalAttrs`?

Reply:
FWIW I based this upon other packages that has this. But you're right, this makes no sense.